docs/index.md

```diff
@@ -16,57 +16,78 @@ OpenVINO™ toolkit:
 
-You can use OpenVINO™ Tuning Utilities with the Inference Engine to trial and test inference on your model. The Benchmark utility uses an input model to run iterative tests for throughput or latency measures, and the [Cross Check Utility](../inference-engine/tools/cross_check_tool/README.md) compares performance of differently configured inferences. 
-
-For a full browser-based studio integrating these other key tuning utilities, try the [Deep Learning Workbench](@ref workbench_docs_Workbench_DG_Introduction).
-![](img/OV-diagram-step3.png)
+You can use OpenVINO™ Tuning Utilities with the Inference Engine to trial and test inference on your model. The [Benchmark utility](../inference-engine/tools/benchmark_tool/README.md) uses an input model to run iterative tests for throughput or latency measures, and the [Cross Check Utility](../inference-engine/tools/cross_check_tool/README.md) compares performance of differently configured inferences. 
 
 OpenVINO™ toolkit includes a set of [inference code samples](IE_DG/Samples_Overview.md) and [application demos](@ref omz_demos) showing how inference is run and output processed for use in retail environments, classrooms, smart camera applications, and other solutions.
 
 OpenVINO also makes use of open-source and Intel™ tools for traditional graphics processing and performance management. Intel® Media SDK supports accelerated rich-media processing, including transcoding. OpenVINO™ optimizes calls to the rich OpenCV and OpenVX libraries for processing computer vision workloads. And the new DL Streamer integration further accelerates video pipelining and performance.
 
+Besides of the [Post-training Optimization Tool](@ref pot_README) which can be used to quantize the model, the [Neural Network Compression Framework (NNCF)](https://github.com/openvinotoolkit/nncf) can be used for model fine-tuning improving INT8 quantization or even applying more aggressive compression methods such as filter pruning, sparsity and binarization to further speedup model inference and reduce its footprint.
+
+For a full browser-based studio integrating these other key tuning utilities, try the [Deep Learning Workbench](@ref workbench_docs_Workbench_DG_Introduction).
+
+![](img/OV-diagram-step3.png)
+
 Useful documents for inference tuning:
 * [Inference Engine Developer Guide](IE_DG/Deep_Learning_Inference_Engine_DevGuide.md)
 * [Inference Engine API References](./api_references.html)
 * [Inference Code Samples](IE_DG/Samples_Overview.md)
 * [Application Demos](@ref omz_demos)
 * [Low Precision Optimization Guide] (@ref pot_docs_LowPrecisionOptimizationGuide)
-* [Deep Learning Workbench Guide](@ref workbench_docs_Workbench_DG_Introduction)
```

Comment:
suggest to left DLWB here, as POT is available in DLWB. I see you add DLWB in step1, but from my perspective we could attract customers to dlwb during pot topic, as it ease customer experience

Reply:
Ok, let's keep it as it was before, even though I think it's useless to put DLWB everywhere.